miner/worker.go

```diff
@@ -877,6 +893,15 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 
 	w.updateSnapshot()
 
+	// Short circuit if this validator is not the proposer of the round/view
+	if !isProposer {
+		log.Info("Tong: Is not a proposer, committing empty block")
+		istanbulEmptyBlockCommit()
```

Comment:
Instead of an empty block submitted to seal, it should be a `nil` block.

Reply:
This comment is irrelevant now, given this comment:  https://github.com/celo-org/celo-blockchain/pull/1102#discussion_r454040327